src/helpers.js

```diff
@@ -1,18 +1,22 @@
 /* eslint no-console:0 */
 const {allowUnsafeNewFunction} = require('loophole')
 const formatPrettierESLint = require('prettier-eslint')
+const minimatch = require('minimatch')
 
 module.exports.format = format
 module.exports.formatOnSaveIfEnabled = formatOnSaveIfEnabled
 
-function format(event, options = {ignoreSelection: false}) {
+function format(
+  event,
+  options = {ignoreSelection: false},
+  editor = atom.workspace.getActiveTextEditor(),
+) {
   /* eslint complexity:[2, 7] */
-  const editor = atom.workspace.getActiveTextEditor()
   if (!editor) {
     return
   }
 
-  const {buffer: {file: {path: filePath} = {}} = {}} = editor
+  const filePath = getCurrentFilePath(editor)
```

Comment:
Moved to a helper below because I also need the file path in `formatOnSave` to check the file against the globs.